src/components/table/index.tsx

```diff
@@ -8,6 +8,11 @@ import { Component, Element, Event, EventEmitter, h } from "@stencil/core"
 export class Table {
 	@Element() element: HTMLSmoothlyTableElement
 	@Event() loadMore: EventEmitter<void>
+	@Listen("expansionLoaded")
```

Comment:
Event names follow present-form (not passed), e.g. `click`, `scroll`, `keydown`, `hover`, `expansionOpen`
```suggestion
	@Listen("expansionLoad")
```